pkg/apis/networking/validation/validation.go

```diff
@@ -315,3 +381,14 @@ func validateIngressBackend(backend *networking.IngressBackend, fldPath *field.P
 	allErrs = append(allErrs, apivalidation.ValidatePortNumOrName(backend.ServicePort, fldPath.Child("servicePort"))...)
 	return allErrs
 }
+
+// defaultBackendFieldName returns the name of the field used for defaultBackend
+// in the provided GroupVersion.
+func defaultBackendFieldName(gv schema.GroupVersion) string {
```

Comment:
As much as I like this function, it should probably be left for a defaultBackend implementation in v1.